src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_3001_4000.cpp

```diff
@@ -561,7 +561,8 @@ GUI_TEST_CLASS_DEFINITION(test_3139) {
     GTMenu::clickMainMenuItem(os, {"File", "Open as..."});
     GTUtilsTaskTreeView::waitTaskFinished(os);
 
-    GTWidget::findWidget(os, "msa_editor_sequence_area");
+    QWidget* seqArea = GTUtilsMSAEditorSequenceArea::getSequenceArea(os, 0);
+    CHECK_SET_ERR(nullptr != seqArea, "MSA Editor isn't opened.!");
```

Comment:
null-checks are not needed. UI test getters should never return null unless asked with FindOptions parameter